src/test/java/org/jenkinsci/plugins/workflow/cps/CpsFlowExecutionTest.java

```diff
@@ -530,14 +530,15 @@ public boolean isAllowed(String groovyResourceUrl) {
         }
     }
 
-    @Issue("JENKINS-45327")
+    @Issue({ "JENKINS-45327", "JENKINS-68849" })
```

Comment:
The issues are not directly related, but it seemed simplest to just have one test covering obscure issues related to restarts and `EnvActionImpl`.